File: zapiski/primerjava.ml
(* OCaml *)
let sez = [1; 2; 3; 4]
let sez2 = 0::sez
(* sez2 kaže na glavo 0, ki kaže naprej na sez. sez2 je narejen v O(1) času *)

let nov_sez = List.rev sez
(* Ustavri nov list, ki ima elemente v obratnem vrstnem redu *)

File: zapiski/primerjava.py
# Python
sez = [1, 2, 3, 4]

sez2 = [0] + sez
# V pomnilniku se ustvari popolnoma nov seznam v O(n) času

ni_nov_sez = sez.reverse()
# Seznam sez se spremeni na mestu. Ne moremo več dostopati do seznama kot je bil prej. ni_nov_sez ima vrednost None